conf/services/customs-financials.cy.yml

```diff
@@ -0,0 +1,36 @@
+serviceName: Bwrw golwg dros eich cyfrifon ariannol ar gyfer y tollau
+serviceHeaderName: Bwrw golwg dros eich cyfrifon ariannol ar gyfer y tollau
```

Comment:
`serviceHeaderName` is no longer used and can be deleted.

Reply:
removed serviceHeaderName